modules/weko-search-ui/weko_search_ui/static/js/weko_search_ui/app.js

```diff
@@ -334,12 +355,48 @@ function itemExportCtrl($scope, $rootScope, $http, $location) {
           }
         });
       })
+      exportBibtex=document.getElementById("export_format_radio_bibtex").checked
+      if (exportBibtex) {
+        let invalidBibtexRecordIds = $scope.validateBibtexExport(Object.keys(export_metadata));
+        if (invalidBibtexRecordIds.length > 0) {
+          $scope.showErrMsgBibtex(invalidBibtexRecordIds);
+        }
+      }
       $('#record_metadata').val(JSON.stringify(export_metadata));
       $('#export_items_form').submit();  // Submit form and let controller handle file making
     }
     $('#item_export_button').attr("disabled", false);
   }
 
+  $scope.validateBibtexExport = function (record_ids) {
+    request_url = '/items/validate_bibtext_export';
+    var data = { record_ids: record_ids }
+    invalidRecordIds = []
```

Comment:
Variable invalidRecordIds implicitly declared.
Please insert "var" or "let" for invalidRecordIds.